ql/src/test/queries/clientpositive/having.q

```diff
@@ -18,4 +19,7 @@ EXPLAIN SELECT key, max(value) FROM src GROUP BY key HAVING max(value) > "val_25
 SELECT key, max(value) FROM src GROUP BY key HAVING max(value) > "val_255";
 
 EXPLAIN SELECT key, COUNT(value) FROM src GROUP BY key HAVING count(value) >= 4;
-SELECT key, COUNT(value) FROM src GROUP BY key HAVING count(value) >= 4;
\ No newline at end of file
+SELECT key, COUNT(value) FROM src GROUP BY key HAVING count(value) >= 4;
+
+EXPLAIN CBO SELECT count(value) as c, max(key) as m from src GROUP BY key HAVING c > 3 and m > 0;
+SELECT count(value) as c, max(key) as m from src GROUP BY key HAVING c > 3 and m > 0;
```

Comment:
Since we are talking about `max(key)` does the `m > 0` condition filter anything? If not then maybe changing to `m > 400` would be a safer choice.